### PR TITLE
OSSMDOC-607: Check Jaeger modules for content type attribute.

### PR DESCRIPTION
**Pull request by JStickler**
Version(s): 4.6 - 4.11

Issue: [OSSMDOC-607 ](https://issues.redhat.com/browse/OSSMDOC-607)

Link to docs preview: no preview, all changes are to file metadata

Additional information:
Peer review: Srivaralakshmi

### CODE REVIEW

File: modules/jaeger-deployment-best-practices.adoc [hunk 1]
```diff
@@ -2,11 +2,9 @@
 This module included in the following assemblies:
 *  /jaeger/jaeger_install/rhbjaeger-deploying.adoc
 ////
-
+:pantheon-module-type: CONCEPT
```
Comment (Srivaralakshmi) [inline modules/jaeger-deployment-best-practices.adoc:5]:
```suggestion
:_content-type: CONCEPT
```
Global comment for similar occurances.

**Question:** 

- Any particular reason why this is different from what is suggested here: https://docs.engineering.redhat.com/pages/viewpage.action?spaceKey=JUP&title=Jupiter+Source+files+readiness+Checklist#JupiterSourcefilesreadinessChecklist-Ifyouarenotadheringtonamingconventions ?

File: modules/jaeger-features.adoc [hunk 2]
```diff
@@ -1,10 +1,10 @@
 ////
-This CONCEPT module included in the following assemblies:
+This module included in the following assemblies:
```
Comment (Srivaralakshmi) [inline modules/jaeger-features.adoc:2]: Refer to this guideline: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata

Suggestion: Even if it is just comment out, it is good to follow what is prescribed as per the guideline as it ensures consistency throughout the files. (Global comment)

File: modules/jaeger-deploy-default.adoc [hunk 3]
```diff
@@ -1,8 +1,8 @@
 ////
-This PROCEDURE module included in the following assemblies:
+This module included in the following assemblies:
 - rhbjaeger-deploying.adoc
 ////
-
+:_content-type: PROCEDURE
```
Comment (Srivaralakshmi) [inline modules/jaeger-deploy-default.adoc:5]: Refer to this guideline example: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata

Observation: I am seeing a line space before the content-type attribute in the guideline example. Please follow this for all the files (global comment).

File: modules/jaeger-install-overview.adoc [hunk 4]
```diff
@@ -1,7 +1,8 @@
 ////
-This CONCEPT module included in the following assemblies:
+This module included in the following assemblies:
 - rhbjaeger-installation.adoc
 ////
+:pantheon-module-type: CONCEPT
```
Comment (Srivaralakshmi) [inline modules/jaeger-install-overview.adoc:5]: The content type for the file. Replace <TYPE> with the actual type of the module, CONCEPT, REFERENCE, or PROCEDURE. Place this attribute before the anchor ID or, if present, the conditional that contains the anchor ID.

Refer to the guideline: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata

Follow the same style.

**Review by Srivaralakshmi — CHANGES_REQUESTED** [review]
@JStickler: PTAL at the suggestions. Thanks!

File: modules/jaeger-deployment-best-practices.adoc (hunk 1 above)
Comment (JStickler) [reply modules/jaeger-deployment-best-practices.adoc:5]: As far as I know, the current standard is `:_content-type: CONCEPT`
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#content-type-attributes

**Pull request merged**